Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -190,33 +190,33 @@ void TextBoxPainter<TextBoxPath>::paintBackground()
 }
 
 template<typename TextBoxPath>
-void TextBoxPainter<TextBoxPath>::paintCompositionForeground()
+void TextBoxPainter<TextBoxPath>::paintCompositionForeground(StyledMarkedText& markedText)
```

Comment:
can this take `const StyledMarkedText&`?

Reply:
oh wait maybe it doesn't even mutate it anymore